perl/lib/Seq/Role/Message.pm

```diff
@@ -208,7 +208,7 @@ sub publishMessage {
   my $timeSinceLastInteraction = time() - $lastPublisherInteractionTime;
   if ( $timeSinceLastInteraction >= $PUBLISHER_ACTION_TIMEOUT ) {
     say
-      "Attempting to reconnect progress publisher because time since last interaction is $timeSinceLastInteraction seconds.";
+      "Attempting to reconnect to publisher in publishMessage because time since last interaction is $timeSinceLastInteraction seconds.";
```

Comment:
We specify where we're calling the publisher to be more specific in logs; otherwise we see sometimes 2 "Attempting to reconnect" back to back.